src/components/Banner/Banner.mdx

```diff
@@ -25,6 +25,7 @@ import { changelogData } from "./bannerChangelogData";
 - {<Link href="#with-html-content" target="_self">With HTML Content</Link>}
 - {<Link href="#custom-color" target="_self">Custom Color</Link>}
 - {<Link href="#dismissible" target="_self">Dismissible</Link>}
```

Comment:
On line 15, update the latest version from `3.2.0` to `"Prerelease"`. It's our pattern for showing that this component is undergoing updates.